src/tests/miscellaneous/mock-adb/app/bin.js

```diff
@@ -58,6 +58,10 @@ async function main() {
     const outputLogsDir = path.join(path.dirname(process.execPath), 'logs', currentContext);
     fs.mkdirSync(outputLogsDir, { recursive: true });
 
+    const testLogsDir = path.join(outputLogsDir, 'testLogs');
+    fs.rmdirSync(testLogsDir, { recursive: true });
+    fs.mkdirSync(testLogsDir);
```

Comment:
Mock-adb gets called several times during an E2E test. If I'm reading this correctly, each call will delete all previous log files. That might make sense in specific scenarios, but I'm not sure it does in the general scenario. Would it make sense to have this behavior controlled by the config file? The default behavior might be to retain the logs, but E2E tests could specify an option that would delete on each call

Reply:
Good catch! Updated bin.js to no longer clear the directory. Instead, added some helper methods to `AppController` to handle this.